src/osd/OSDMap.cc

```diff
@@ -5904,6 +5905,27 @@ void OSDMap::check_health(health_check_map_t *checks) const
       d.detail.swap(nearfull_detail);
     }
   }
+
+  // POOL_PG_NUM_NOT_POWER_OF_TWO
+  if (cct->_conf.get_val<bool>("mon_warn_on_pool_pg_num_not_power_of_two")) {
```

Comment:
@liewegas  Did you see my comment above? I think you can directly  use the `g_conf ` wrapper here, e.g., line 5829.

or there must be something that I am missing...

Reply:
I've added another commit that switches the g_conf() users to use the explicit cct.  Using the explicit cct is preferable, and easy in this case since only check_health() ever needs it.